iidm/iidm-api/src/main/java/com/powsybl/iidm/network/VoltageAngleLimit.java

```diff
@@ -0,0 +1,65 @@
+/**
+ * Copyright (c) 2023, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+package com.powsybl.iidm.network;
+
+/**
+ * @author Luma Zamarreño <zamarrenolm at aia.es>
+ * @author José Antonio Marqués <marquesja at aia.es>
+ */
+public interface VoltageAngleLimit extends OperationalLimits {
+
+    public enum FlowDirection {
+        FROM_TO,
+        TO_FROM,
+        BOTH_DIRECTIONS
+    }
+
+    @Override
+    default LimitType getLimitType() {
+        return LimitType.VOLTAGE_ANGLE;
+    }
+
+    /**
+     * Get the id
+     */
+    default String getId() {
+        return getTerminalFrom().getConnectable().getId();
```

Comment:
Why do you put that? You just introduce a name or an id given at limit construction?

Reply:
Used to fill `LimitViolation`. We removed it and we directly assign to `LimitViolation.subjectId` the connectable id associated to terminal from. `SubjectId` must be a connectable id.

Reply:
Where do we check that it is a connectable? I don't see the check in the code. The side can be `null`.

Reply:
In this version, `VoltageAngleLimit` does not have an `id` attribute. We assign to `LimitViolation.subjectId` the connectable id associated to terminal from. In the VoltageAngleLimit adder we verify that terminal from and terminal to are well defined.